sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-generation.service.ts

```diff
@@ -332,8 +439,35 @@ export class ECertGenerationService {
         .reduce((previousValue, currentValue) => {
           return previousValue + currentValue.effectiveAmount;
         }, 0);
+      // Calculate total BC grants subtracted due to restriction.
```

Comment:
Is it not supposed to be taken care of during the effective value calculation?

Reply:
I added it here because the total bc grant itself was here at this stage, after the effective calculation
![image](https://user-images.githubusercontent.com/77353155/232910387-cdc88b01-b8ab-41bf-b9a7-ef650c044bff.png)

Reply:
https://github.com/bcgov/SIMS/pull/1886#discussion_r1171973155